src/Settings/Model/GeneralSettings.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Settings\Model;
+
+use MyParcelNL\Pdk\Base\Model\Model;
+
+/**
+ * @property string $apiKey
+ * @property bool   $apiLogging
+ * @property bool   $shareCustomerInformation
+ * @property bool   $useSecondAddressField
+ * @property bool   $conceptShipments
+ * @property bool   $pps
+ * @property string $priceType
+ */
+class GeneralSettings extends Model
+{
+    protected $attributes = [
+        'apiKey'                   => null,
+        'apiLogging'               => false,
+        'shareCustomerInformation' => false,
+        'useSecondAddressField'    => false,
+        'conceptShipments'         => true,
+        'pps'                      => false,
```

Comment:
Voor de zekerheid, zijn we het allemaal eens dat we dit pps noemen? Of moeten we van de gelegenheid gebruik maken om een betere naam te introduceren? (ik vind het eigenlijk niet bijzonder duidelijk (meer))

Reply:
nee, we hadden gekozen om het iets in de richting van "order mode" te noemen, zo heet het ook in de api